src/components/Navbar/DropDownItem.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const DropDownItem = (props) => {
+  return (
+    <Link className="nav--dropdown__link" to={props.linkTo}>
+      <li>{props.name}</li>
+    </Link>
+  );
+};
+
+export default DropDownItem;
```

Comment:
🤔 idk if this should be a separate component

Reply:
true, it was too simple. refactored and deleted